mlflow/utils/databricks_utils.py

```diff
@@ -162,6 +164,15 @@ def is_in_databricks_job():
         return False
 
 
+def is_in_databricks_model_serving_environment():
+    # choose 2 fairly specific environment variable that should always be configured
```

Comment:
Beautiful!!! Thanks @victorsun123 !

Reply:
@wentinghu / @sueann - Can we rely on these to always exist? 

My preference would be to introduce a new `IS_DATABRICKS_MODEL_SERVING` env than relying on flags that may get changed or removed in the future WDYT?